docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,106 @@
+---
+title: What to know before onboarding
+sidebar_position: 8
+---
+# What maintainers should know before adopting Packit for downstream release automation 
+
+## Responsibilities
+
+Packit is designed to **assist** maintainers, not to **replace** them.
+
+Maintainers retain key responsibilities:
+- reviewing and merging Packit-created pull requests
+- maintaining package quality
+- ensuring updates don't break dependant packages
+
+:::tip `version_update_mask`
+
+The default Packit onboarding configuration enables automated releases only for `rawhide`. 
+
+When extending the automation to branched Fedoras and EPELs, consider using [`version_update_mask`](https://packit.dev/docs/configuration#version_update_mask) to prevent introducing breaking changes into stable distributions.
+
+:::
+
+:::warning Missing notifications
+
+Pull requests, Koji builds and Bodhi updates are created by Packit FAS account, making it the default notification recipient.
+
+To stay informed about Packit-triggered activities in the Fedora ecosystem, [configure notification settings accordingly](https://packit.dev/docs/fedora-releases-guide/fedora-notifications).
+
+There is an [open issue](https://github.com/packit/packit-service/issues/2404) for Bodhi update notifications.
+
+:::
+
+## Monitoring and observability
+
+Track *Packit Service* activity through:
+- the pull request updates, and 
+- the [Packit Dashboard](https://dashboard.packit.dev/).
+
+Packit Service responds to every valid command in pull requests. Missing response indicates:
```

Comment:
maybe we could link https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding#retriggering here for the commands?

Reply:
ok I see it is linked in the following section, in that case maybe this sentence with the 2 points could be moved there as well?

Reply:
I meant it as a monitoring indication (you can understand packit is reacting waiting for the response). For this reason I would leave it here. But I linked again all the commands.

Reply:
ok, makes sense!